app/_includes/docs-sidebar.html

```diff
@@ -19,6 +61,7 @@
   {% endif %}
 
   <ul class="accordion-container">
```

Comment:
1. After going back and forth with my teammate, we both agreed that the new styling for the sidebar feels cramped/busy, especially with the icons. Previously, the sidebar stretched too far down the page, but we also had a huge header. Without the header, the previous styling works quite well (see screenshot in previous comment). Can you increase the white space for each top-level item?

2. Can we not use the Roboto font here? It’s too jarring with the content text - let’s stick to using it only as a display font (in h1/title, in breadcrumbs, in dropdowns).
Bring down font weight to 500 or so? 

3.  Sidebar is too wide and content too narrow on a screen wider than 1600px, looks like there's some responsiveness issue:

At 1599px:
![Screen Shot 2021-03-08 at 3 24 40 PM](https://user-images.githubusercontent.com/54370747/110395270-b478ef80-8022-11eb-819a-9e7036528f30.png)

At 1600px:
![Screen Shot 2021-03-08 at 3 24 52 PM](https://user-images.githubusercontent.com/54370747/110395327-cb1f4680-8022-11eb-8c59-431426b66a9c.png)